src/applications/public-outreach-materials/sass/public-outreach-materials.scss

```diff
@@ -0,0 +1,111 @@
+$formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
+
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-nav-sidebar";
+
+.outreach-asset-form-filters {
+  padding: 15px;
+  background: $color-gray-lightest;
+  max-width: 95%;
+  margin-bottom: 25px;
+  @media (max-width: $medium-screen) {
+    .vads-l-col--6,
+    .vads-l-col--5,
+    .vads-l-col--4 {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+  }
+  .vads-l-grid-container {
```

Comment:
I think you can use `vads-l-grid-container--full` instead of this

Reply:
:+1: @jbalboni